src/CONST.ts

```diff
@@ -1370,6 +1370,9 @@ const CONST = {
         MAKE_REQUEST_WITH_SIDE_EFFECTS: 'makeRequestWithSideEffects',
     },
 
+    MAP_PADDING: 50,
```

Comment:
This `MAP_PADDING` value caused the following issue:
- https://github.com/Expensify/App/issues/46296

Because the value was too high, on certain narrow layout devices (mobile) the value caused the app to crash when submitting a distance request. The issue was fixed by decreasing the value to `32` and removing / unifying the `MAP_PADDING` into using `MAPBOX.PADDING` variable instead. More details can be found in the issue / [proposal](https://github.com/Expensify/App/issues/46296#issuecomment-2252879181).